Python/bytecodes.c

```diff
@@ -727,8 +728,38 @@ dummy_func(
         #endif
         }
 
+       op(_GUARD_BINARY_OP_EXTEND, (left, right -- left, right)) {
```

Comment:
```suggestion
       op(_GUARD_BINARY_OP_EXTEND, (descr/4, left, right -- left, right)) {
```
Let the code generator do the offset calculation if you can